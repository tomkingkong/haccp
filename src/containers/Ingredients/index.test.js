import React from 'react';
import { shallow } from 'enzyme';

import { Ingredients } from '.';

describe('Ingredients', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Ingredients ingredients={[]} addIngredient={() => jest.fn()}/>);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});