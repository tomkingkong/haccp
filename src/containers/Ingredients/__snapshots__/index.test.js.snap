// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ingredients should match snapshot 1`] = `
<div
  className="ingredients"
>
  <h2>
    Ingredients
  </h2>
  <IngredientsForm
    handleNewIngredient={[Function]}
    ingredients={Array []}
  />
  <button
    className="save-continue"
    onClick={[Function]}
  >
    save & continue
  </button>
</div>
`;
